pkg/apis/nsx.vmware.com/v1alpha1/nsxserviceaccount_types.go

```diff
@@ -0,0 +1,99 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
```

Comment:
Why we have these dup files?

Reply:
You mean file pkg/apis/nsx.vmware.com/v1alpha1/nsxserviceaccount_types.go and pkg/apis/v1alpha1/nsxserviceaccount_types.go? These two files are duplicated?

Reply:
Yes. Maybe whole `pkg/apis/nsx.vmware.com` is duplicated?

Reply:
As I know, these files in pkg/apis/nsx.vmware.com folder is used when nsx operator is used as a module  @zhengxiexie please confirm this?

Reply:
Yes

Reply:
Any method to make these 2 folder synchronized, or at least notify developers to sync it manually?

Reply:
`make generated` already sync

Reply:
@zhengxiexie  I found the files in pkg/apis/nsx.vmware.com/v1alpha1/2 has some differences with the files in pkg/apis/v1alpha1/2, Could you help check which one is latest version?  if necessary, we might need to a new PR to sync these two folders.

Reply:
@gran-vmv  https://github.com/vmware-tanzu/nsx-operator/pull/402 yes, this PR is to sync files between two places.  These files are synced by make generated